src/components/dialogs/parameters/sensi/sensi-injections-set.js

```diff
@@ -0,0 +1,173 @@
+/**
```

Comment:
It is almost the same code as the previous file (sensi-hvdcs.js). Can we refactor these 2 files ?

Reply:
done